webrtc-stats.html

```diff
@@ -1218,6 +1218,16 @@
              double                        roundTripTime;
              double                        availableOutgoingBitrate;
              double                        availableIncomingBitrate;
+             unsigned long long            requestsReceived;
+             unsigned long long            requestsSent;
+             unsigned long long            responsesReceived;
+             unsigned long long            responsesSent;
+             unsigned long long            retransmissionsSent;
+             double                        consentTotalRoundTripTime;
```

Comment:
If this is for the roundtrip before we receive consent, isn't there by definition one RTT sample (since once we receive one response, we have consent)?  How useful is that really?  I think we should just have one RTT value, not pre- and post-consent values.